### ISSUE TITLE
Cannot resolve 'IntType'

### ISSUE THREAD

**Issue by LinkTechTips** [xCollateral/VulkanMod#141]
![7$Q92$I10$V8JLKYWFI9K78](https://user-images.githubusercontent.com/88884182/200385452-656a3a6a-94d1-4b98-9185-e0225be0be72.png)
About source code:
[VulkanMod/blob/main/src/main/java/net/vulkanmod/mixin/render/IntTypeMixin.java#L7](https://github.com/xCollateral/VulkanMod/blob/main/src/main/java/net/vulkanmod/mixin/render/IntTypeMixin.java#L7)
[VulkanMod/blob/main/src/main/java/net/vulkanmod/mixin/render/IntTypeMixin.java#L14](https://github.com/xCollateral/VulkanMod/blob/main/src/main/java/net/vulkanmod/mixin/render/IntTypeMixin.java#L14)
[VulkanMod/blob/main/src/main/java/net/vulkanmod/mixin/render/IntTypeMixin.java#L15](https://github.com/xCollateral/VulkanMod/blob/main/src/main/java/net/vulkanmod/mixin/render/IntTypeMixin.java#L15)

**Comment by LinkTechTips** [comment]
`F:\VulkanMod\src\main\java\net\vulkanmod\mixin\render\IntTypeMixin.java:7: ����: Mixin has no targets
@Mixin(VertexFormat.IntType.class)
^`

**Comment by MinhCrafters** [comment]
You're probably using 1.19.x to edit the source code.
Change to 1.18.2 by editing the `build.gradle` file, because this branch is based on 1.18 mappings.

**Comment by LinkTechTips** [comment]
> You're probably using 1.19.x to edit the source code. Change to 1.18.2 by editing the `build.gradle` file, because this branch is based on 1.18 mappings.

Yes, I am using 1.16.5 to edit the source code. Does 1.16.5 support it?

**Comment by MinhCrafters** [comment]
> Yes, I am using 1.16.5 to edit the source code. Does 1.16.5 support it?

Yes, it does. Just under a different name and stuff.

**Comment by LinkTechTips** [comment]
Okay, I think I should know what to do next. Thank you for your technical support
